source/client-side-encryption/tests/README.rst

```diff
@@ -2693,302 +2694,343 @@ Create a ClientEncryption object named ``clientEncryption`` with these options:
+
+   // Convert 200 to the encrypted field type
+   { "_id": 3, "encrypted<Type>": 200 }
+
 
-Case 5: can run an aggregation expression inside $expr 
+Case 5: can run an aggregation expression inside $expr
 ``````````````````````````````````````````````````````
-Use ``clientEncryption.encryptExpression()`` to encrypt this query: 
+Use ``clientEncryption.encryptExpression()`` to encrypt this query:
 
 .. code:: javascript
 
-   {'$and': [ { '$lt': [ '$encrypted<Type>', 30 ] } ] } }
+   // Convert 30 to the encrypted field type
+   { "$and": [ { "$lt": [ "$encrypted<Type>", 30 ] } ] } }
 
-Use the matching ``RangeOpts`` listed in `Test Setup: RangeOpts`_ and these ``EncryptOpts`` to encrypt the query:
+Encrypt using the following ``EncryptOpts``:
 
 .. code:: typescript
 
    class EncryptOpts {
-      keyId : <key1ID>
+      keyId : <key1ID>,
       algorithm: "RangePreview",
       queryType: "rangePreview",
-      contentionFactor: 0
+      contentionFactor: 0,
+      rangeOpts: <RangeOpts for Type>,
    }
 
 Store the result in ``findPayload``.
 
-Use ``encryptedClient`` to run a "find" operation on the ``db.explicit_encryption`` collection with the filter ``{'$expr' :  <findPayload> }`` and sort the results by ``_id``.
+Use ``encryptedClient`` to run a "find" operation on the ``db.explicit_encryption`` collection with the filter ``{ "$expr": <findPayload> }`` and sort the results by ``_id``.
+
+Assert the following two documents are returned:
+
+.. code:: javascript
+
+   // Convert 0 and 6 to the encrypted field type
+   { "_id": 0, "encrypted<Type>": 0 }
+   { "_id": 1, "encrypted<Type>": 6 }
 
-Assert that these two documents ``{ "encrypted<Type>": 0 }, { "encrypted<Type>": 6 }`` are returned.
 
 Case 6: encrypting a document greater than the maximum errors
 `````````````````````````````````````````````````````````````
 This test case should be skipped if the encrypted field is ``encryptedDoubleNoPrecision`` or ``encryptedDecimalNoPrecision``.
 
-Use ``clientEncryption.encrypt()`` to try to encrypt the value 201 with the matching ``RangeOpts`` listed in `Test Setup: RangeOpts`_ and these ``EncryptOpts``:
+Use ``clientEncryption.encrypt()`` to encrypt the value 201. Ensure the type matches that of the encrypted field.
+
+Encrypt using the following ``EncryptOpts``:
 
 .. code:: typescript
 
    class EncryptOpts {
-      keyId : <key1ID>
+      keyId : <key1ID>,
       algorithm: "RangePreview",
-      contentionFactor: 0
+      contentionFactor: 0,
+      rangeOpts: <RangeOpts for Type>,
    }
 
-Ensure 201 matches the type of the encrypted field. The error should be raised because 201 is greater than the maximum value in ``RangeOpts``.
+Assert that an error was raised because 201 is greater than the maximum value in ``RangeOpts``.
 
-Assert that an error was raised.
 
-Case 7: encrypting a document of a different type errors 
-````````````````````````````````````````````````````````
+Case 7: encrypting a value of a different type errors
```

Comment:
The original prose test was unclear if this was testing insertion (e.g. wrong type in the `encrypted<Type>` field) or direct encryption of a value.

I assumed the latter based on the [libmongoc test](https://github.com/mongodb/mongo-c-driver/commit/7a6b6369eb918bc2698fd6dc3aeab12845176a36#diff-d363a5539f2fbabfadf2d7cc6cdb96a203fb63f2caa31bae731c3e12d97b204cR4333).

Is there any concern other drivers got this wrong and are instead testing with document insertion? Should we have two separate prose tests?